MusicNotationKit/MusicNotationKit/Measure.swift

```diff
@@ -46,12 +46,36 @@ public struct Measure: ImmutableMeasure, Equatable {
         noteCount = immutableMeasure.noteCount
         recomputeNoteCollectionIndexes()
     }
-
+	
+	public func note(at index: Int, inSet setIndex: Int) throws -> Note {
+		let noteCollectionIndex = try noteCollectionIndexFromNoteIndex(index, inSet: setIndex)
```

Comment:
Can you rename this method to have the from as part of the first parameter name? Similar to my comment below.